platforms/documentation/docs/src/docs/userguide/userguide_single.adoc

```diff
@@ -190,15 +189,14 @@ include::dependency_caching.adoc[leveloffset=+2]
 
 == UNDERSTANDING DEPENDENCY RESOLUTION
 
-include::dependency_resolution_model.adoc[leveloffset=+2]
-include::variant_model.adoc[leveloffset=+2]
+include::dependency_resolution.adoc.adoc[leveloffset=+2]
+include::variant_aware_resolution.adoc[leveloffset=+2]
 include::component_capabilities.adoc[leveloffset=+2]
 include::variant_attributes.adoc[leveloffset=+2]
 
 == CONTROLLING DEPENDENCY RESOLUTION
 
-include::dependency_resolution_basics.adoc[leveloffset=+2]
-include::dependency_graph_resolution.adoc[leveloffset=+2]
+include::graph_resolution.adoc[leveloffset=+2]
```

Comment:
I did not include `dependency_resolution.adoc` which corresponds to `dependency_resolution_basics.adoc`, since it is already included in L192.

https://github.com/gradle/gradle/blob/1ceb6a55a8b9f772540ea2655f7ec71cf71e773d/platforms/documentation/docs/src/docs/userguide/redirects/dependency_resolution_basics.adoc?plain=1#L2

https://github.com/gradle/gradle/blob/1ceb6a55a8b9f772540ea2655f7ec71cf71e773d/platforms/documentation/docs/src/docs/userguide/redirects/dependency_graph_resolution.adoc?plain=1#L2